docs/_includes/navbar.html

```diff
@@ -24,15 +24,15 @@
         We might be on an externally hosted documentation site.
         Please keep the site.FLINK_WEBSITE_URL below to ensure a link back to the Flink website.
         {% endcomment %}
-	<a href="{{ site.FLINK_WEBSITE_URL }}index.html" title="Home">
+	<a href="{{ site.FLINK_WEBSITE_URL }}/index.html" title="Home">
 	  <img class="hidden-xs hidden-sm img-responsive"
-	       src="{{ site.baseurl }}img/logo.png" alt="Apache Flink Logo">
+	       src="{{ site.baseurl }}/img/logo.png" alt="Apache Flink Logo">
 	</a>	
 	<div class="row visible-xs">
 	  <div class="col-xs-3">
-	    <a href="{{ site.baseurl }}index.html" title="Home">  
+	    <a href="{{ site.baseurl }}/index.html" title="Home">
```

Comment:
Yes, we can set in it the _config.yml. I would not change it now because it has some other implications. For example, local testing of the website will be more complicated because all links will point to the online version. We can open a separate issue.